### DISCUSSION TITLE
Problems with the startup of the docker

### DISCUSSION THREAD

**Discussion by medini-sr** [larics/icuas24_competition#24] [General]
![Screenshot from 2024-01-08 17-51-33](https://github.com/larics/icuas23_competition/assets/117924750/e514a516-2404-49cf-b7bb-c1d4799e2906)
I built the docker and when I try to run it, I got this.Could you please tell me how to fix this.

**Comment by fpetric** [comment]
This looks like Docker installation issue. Did you install Docker desktop or Docker ce?
